docs/content/guides/upgrade-and-migration/migrating-from-15.3-to-16.0/migrating-from-15.3-to-16.0.md

```diff
@@ -0,0 +1,416 @@
+---
+id: migrating-15.3-to-16.0
```

Comment:
This `id` should match [the spec](https://github.com/handsontable/handsontable/blob/develop/docs/README-EDITING.md#frontmatter). Basically, it should be an 8-character random key. Different for JS, React, and Angular pages.